### DISCUSSION TITLE
Can I have remote `smtp.template_override_path`?

### DISCUSSION THREAD

**Discussion by hugotiburtino** [ory/kratos#2666] [Q&A]
for example:
```yaml
smtp:
  template_override_path: https://mysite.org/.ory/mail-templates
```

In the [documentation](https://www.ory.sh/docs/kratos/concepts/email-sms#message-templates) it seems not to be possible, although it doesn't exclude it. I've tested and it doesn't work (but, if yes, I may have not tested enough)...

If not, what about adding this feature?

**Comment by vinckr** [comment]
Hey @hugotiburtino 

> In the [documentation](https://www.ory.sh/docs/kratos/concepts/email-sms#message-templates) it seems not to be possible, although it doesn't exclude it.

Generally we document what is possible, not what is not possible - in this case this is possible when self-hosting Kratos. 
From the [reference configuration](https://www.ory.sh/docs/kratos/reference/configuration):

```
  ## Override message templates ##
  #
  # You can override certain or all message templates by pointing this key to the path where the templates are located.
  #
  # Examples:
  # - /conf/courier-templates
  # 
  # Set this value using environment variables on
  # - Linux/macOS:
  #    $ export COURIER_TEMPLATE_OVERRIDE_PATH=<value>
  # - Windows Command Line (CMD):
  #    > set COURIER_TEMPLATE_OVERRIDE_PATH=<value>
  #
  template_override_path: /conf/courier-templates
```

How are you running Ory Kratos?
I have not used this before, I just inline the templates base64 encoded, but it would be good to have if its not possible right now for sure...

**Reply by hugotiburtino** [reply]
Hi @vinckr,

Thanks for you answer. 

To answer your question: we are self-hosting it in a Kubernetes cluster.

But yes, as in the documentation there is only examples with local storage. As you rightly say, you can't documentate everything, only what is possible. So it would be a feature request.
